### PR TITLE
DOC: Blogpost_KD_Week8

### PR DESCRIPTION
**Pull request by deka27**
Summary:
This pull request introduces a blog post for the 9th week of GSOC. It aims to provide insights and reflections for future work as well.

Key Changes:

Added a new blog post file in the posts/2024 directory.

### CODE REVIEW

File: posts/2024/2024_07_22_kaustav_week_8.rst [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+I didn't particularly find anything difficult, the tasks were time consuming but not trouble causing.
+
+Next Week
+---------
+
+Looking ahead to next week, I have several objectives in mind:
+
+1. I plan to continue addressing open issues on the `dipy.org` & `dipy` repository. There are still several tasks that need attention, and I'm eager to keep up the momentum we've built. I have yet to look into which issues I will be tackling next week.
+
+2. For the preparation of next task, I have to look into Docker. I am planning to learn it through next week to be well prepared for the next task which is about github actions.
```
Comment (itellaetxe) [inline posts/2024/2024_07_22_kaustav_week_8.rst:77]: Regarding this, the University of Helsinki has an [incredible (free) course on Docker](https://devopswithdocker.com/). Maybe it helps you learn the tool. It was all I needed to get started with Docker.

**Review by itellaetxe — APPROVED** [review]
Amazing job, @deka27.

Thanks for your contributions, now the repo is tidier and the blog is nicer with the sidebar.

LGTM. Just a small comment, hope you will like it ;)

**Review by skoudoro — APPROVED** [review]
Great Job and huge work!

thank you for all of this. Blog posts looks good to me so merging

File: posts/2024/2024_07_22_kaustav_week_8.rst (hunk 1 above)
Comment (deka27) [reply posts/2024/2024_07_22_kaustav_week_8.rst:77]: Thanks for this. I will definitely require it.

**Pull request merged**